src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1284,11 +1284,15 @@ public LocalDate fromString(String string) {
           textFieldCvrFilePath.setDisable(false);
           buttonCvrFilePath.setDisable(false);
           textFieldCvrFirstVoteCol.setDisable(false);
-          textFieldCvrFirstVoteCol
-                  .setText(String.valueOf(ContestConfig.SUGGESTED_CVR_FIRST_VOTE_COLUMN));
+          textFieldCvrFirstVoteCol.setText("3");
           textFieldCvrFirstVoteRow.setDisable(false);
           textFieldCvrFirstVoteRow
                   .setText(String.valueOf(ContestConfig.SUGGESTED_CVR_FIRST_VOTE_ROW));
+          textFieldCvrBatchCol.setDisable(false);
+          textFieldCvrIdCol.setDisable(false);
+          textFieldCvrIdCol.setText("1");
+          textFieldCvrPrecinctCol.setDisable(false);
+          textFieldCvrPrecinctCol.setText("2");
```

Comment:
I expect most users won't use these -- would it make sense for the default to be blank to signal to users it's okay to leave it blank?